kong/plugins/prometheus/api.lua

```diff
@@ -1,10 +1,15 @@
 local prometheus = require "kong.plugins.prometheus.exporter"
 
+local consumers_schema = kong.db.consumers.schema
 
 return {
+
   ["/metrics"] = {
-    GET = function(self, dao_factory) -- luacheck: ignore 212
-      prometheus.collect()
-    end,
+    schema = consumers_schema, -- not used, could be any schema
```

Comment:
This comment is confusing... The reader is left wondering why we are specifying a schema at all then?